src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -83,9 +95,67 @@
     }
 @end
 
+@private clientStreamingMethodSampleCode(apiMethod)
+    $requests = [$request];
+    
+    // Write data to server and wait for a response
+    $streamingResponse = {@methodCallSampleCode(apiMethod)};
+    $result = $streamingResponse->writeAllAndWait($requests);
```

Comment:
The variable names could be method-specific, but the GAX types are the same for all methods, so they need to be named for the streaming type